SoftLayer/CLI/user/delete.py

```diff
@@ -0,0 +1,30 @@
+"""Delete user."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import helpers
+
+
+@click.command()
+@click.argument('identifier')
+@environment.pass_env
+def cli(env, identifier):
+    """Delete a User.
```

Comment:
To be clear to the customer, I would change the help message to this:

"Sets a user's status to CANCEL_PENDING, which will immediately disable the account, and will eventually be fully removed from the account by an automated internal process"

Reply:
updated, tox analysis should pass too